bin/sdk-cli.js

```diff
@@ -10,41 +10,51 @@ const path = require( 'path' );
 const spawnSync = require( 'child_process' ).spawnSync;
 const yargs = require( 'yargs' );
 
-const buildBlock = argv => {
-	const compiler = path.resolve( __dirname, 'create-scripts/block.js' );
-	const editorScript = path.resolve( __dirname, '../', argv.editorScript );
+/**
+ * Internal dependencies
+ */
+const gutenberg = require( './sdk/gutenberg.js' );
 
-	spawnSync( 'node', [ compiler, editorScript, ( argv.outputDir || '' ) ], {
-		env: {
-			SKIP_FLAG_IMAGES: true,
-		},
-		shell: true,
-		stdio: 'inherit',
-	} );
-};
+// Script name is used in help instructions;
+// pick between `npm run calypso-sdk` and `npx calypso-sdk`.
+// Show also how npm scripts require delimiter to pass arguments.
+const calleeScript = path.basename( process.argv[ 1 ] );
+const scriptName = calleeScript === path.basename( __filename ) ? 'npm run calypso-sdk' : calleeScript;
+const delimit = scriptName.substring( 0, 3 ) === 'npm' ? '-- ' : '';
 
 yargs
-	.scriptName( 'calypso-gutenberg-sdk' )
-	.usage( 'Usage: $0 <command> [options]' )
+	.scriptName( scriptName )
+	.usage( `Usage: $0 <command> ${ delimit }[options]` )
+	.example( `$0 build-gutenberg ${ delimit }--editor-script=hello-dolly.js` )
 	.command( {
-		command: 'build-block',
-		desc: 'Build a block',
+		command: 'build-gutenberg',
+		desc: 'Build a Gutenberg extension',
 		builder: yargs => yargs.options( {
+			'mode': {
```

Comment:
Noting that `mode` arg could as well be in a separate PR to focus this one a bit more since PR has plenty going on already.